### ISSUE TITLE
Abnormal social communication behavior

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#8028]
**Preferred term label:**
Abnormal social communication behavior

**Synonyms**

**Definition (free text, please give PubMed ID)**
_will be discussed at next meeting_ 

**Parent term (use hpo.jax.org/app)**
Abnormal social behavior (HP:0012433)

**Note**
"Abnormal nonverbal communicative behavior" and "Abnormal verbal communicative behavior" should be moved from being children of Abnormal social behavior to being children of this term (Abnormal social communication behavior).

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
@bcoleman96 
Added with preliminary definition
Any abnormal behavior related to communication.
